modules/developer_manual/pages/testing/unit-testing.adoc

```diff
@@ -101,11 +101,14 @@ make test			run all tests
 
 ==== Testing Apps
 
-To run the tests for a specific app with the provided PHPUnit version: 
+To run the tests for a specific app with the provided PHPUnit version:
 
-. Change into the apps directory, in the root directory of your ownCloud installation
-. Then change into the directory of the app that you want to test. 
-. Run the follow command. 
+. Change into the `apps-external` directory, in the root directory of your ownCloud installation.
++
+IMPORTANT: Please *do not* use the `apps` directory.
```

Comment:
The recommendation is to use the default `apps_paths` setting that comes in `config.php` these days.
So you `cd apps-external` and then clone the app from github into there, then "change into the newly cloned directory"